### PR TITLE
Feature/add tabindex to item picker

### PR DESCRIPTION
**Pull request by leonidagji**
### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation, readme, or JSdoc annotations)
- [x] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

This pull request adds the `tabindex` attribute to the up and down arrows in the `six-item-picker` component. This change improves accessibility by allowing users to navigate and interact with the arrows using keyboard controls.

### 📝 Checklist

- [ ] It's submitted to the `main` branch
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] I have updated the documentation accordingly.
- [ ] All tests are passing
- [ ] New/updated tests are included
- [ ] I have updated the changelog "upcoming" section with the changes I contributed

**Other information:**
image
image

### CODE REVIEW

**Review by ibirrer — CHANGES_REQUESTED** [review]
I recommend to make a PR only with the fix for the close icon.  See [comment ](https://github.com/six-group/six-webcomponents/pull/132#discussion_r1391265956)about the item picker.

File: libraries/ui-library/src/components/six-item-picker/six-item-picker.tsx [hunk 1]
```diff
@@ -457,31 +457,137 @@ export class SixItemPicker {
...
+  }
+
+  private upButton!: HTMLDivElement;
+  private downButton!: HTMLDivElement;
+
   render() {
     return (
-      <div part="container" class="item_picker__container">
+      <div
+        part="container"
```
Comment (ibirrer) [inline libraries/ui-library/src/components/six-item-picker/six-item-picker.tsx:560]: We should use the same focus style for all elements. In this case you can simply add the following css to this container:
```css
&:focus-visible {
  outline: var(--six-focus-ring);
  outline-offset: 0;
}
```

File: libraries/ui-library/src/components/six-item-picker/six-item-picker.tsx [hunk 2]
```diff
@@ -457,31 +457,137 @@ export class SixItemPicker {
     return this.value;
   }
 
+  private handleKeyPress(event: KeyboardEvent) {
```
Comment (ibirrer) [inline libraries/ui-library/src/components/six-item-picker/six-item-picker.tsx:460]: `PageDown/Up`, and `Home/End`  are not firing a `six-item-picker-change` event

**Review by ibirrer — CHANGES_REQUESTED** [review]

File: libraries/ui-library/src/components/six-item-picker/six-item-picker.tsx (hunk 2 above)
Comment (leonidagji) [reply libraries/ui-library/src/components/six-item-picker/six-item-picker.tsx:462]: I added the six-item-picker-change event in the changeValue method.

**Review by ibirrer — APPROVED** [review]

**Pull request merged**